airbyte-integrations/connectors/source-amazon-ads/source_amazon_ads/schemas/sponsored_product_targetings.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "properties": {
+    "expression": {
+      "type": "array",
+      "items": { "type": "object", "additionalProperties": true }
+    },
+    "targetId": { "type": "string" },
+    "resolvedExpression": {
+      "type": "array",
```

Comment:
Don't want to be annoying but, some schemas have nullable properties, some doesn't. Not sure if this was the case before, do we consider make all of them nullable? Maybe this is not a requirement anymore, in general?

Reply:
schemas were generated from pydantic, so no changes were introduced, but I agree, already updated to make all nullable.